src/modules/sensors/vehicle_angular_velocity/VehicleAngularVelocity.hpp

```diff
@@ -94,12 +103,23 @@ class VehicleAngularVelocity : public ModuleParams, public px4::WorkItem
 		{this, ORB_ID(sensor_gyro), 2}
 	};
 
+	perf_counter_t _interval_perf{perf_alloc(PC_INTERVAL, MODULE_NAME": interval")};
+
 	matrix::Dcmf _board_rotation;
 
 	matrix::Vector3f _bias{0.f, 0.f, 0.f};
 	matrix::Vector3f _offset{0.f, 0.f, 0.f};
 	matrix::Vector3f _scale{1.f, 1.f, 1.f};
 
+	hrt_abstime _last_publish{0};
+	hrt_abstime _filter_check_last{0};
+	static constexpr const float kINITIAL_RATE_HZ{1000.0f}; /**< sensor update rate used for initialization */
```

Comment:
typo? INITIAL_RATE_HZ

Reply:
I don't recall, but might have been a half-assed partial attempt at following the google c++ style guide. Actually following the google c++ style guide would be `kInitialRateHz`.

Fixing...